packages/ui-library/src/components/internal-components/form-label/index.test.ts

```diff
@@ -14,16 +16,39 @@ const sampleParams: BlrFormLabelType = {
   variant: 'label',
 };
 
+// The label is not creating a shadow root itself, but errors if it is outside
+// of one. Thus, we're creating a helper component for the stories, that wraps it.
+class WrappedBlrLabel extends LitElement {
+  labelProps: BlrFormLabelType;
+
+  protected render() {
+    return html`${BlrFormLabelRenderFunction(this.labelProps)}`;
```

Comment:
minor thing: these render functions already return html``-things. So this wrapping in html-tag-function is not needed